www/main/navigator/search/utils/retrieve-from-db.js

```diff
@@ -1,15 +1,15 @@
-const remote = require('@electron/remote');
-
 import { getFilterOption } from '../../../banidb/realm-search';
 
+const remote = require('@electron/remote');
+
 const { i18n } = remote.require('./app');
 
 export const retrieveFilterOption = async (optionsObj, type) => {
   const idArray = Object.keys(optionsObj).filter(
-    option => option.toLowerCase() !== 'all' && option.toLowerCase() !== 'others',
+    (option) => option.toLowerCase() !== 'all' && option.toLowerCase() !== 'others',
   );
   const retrievedObj = await getFilterOption(type, idArray);
-  const valueObj = Object.assign({}, retrievedObj);
+  const valueObj = { ...retrievedObj };
```

Comment:
@theamanjs veerji, can you confirm if this is correct, the linter did it on its own. I checked the functionality and the app is working fine.